### ISSUE TITLE
3week

### ISSUE THREAD

**Issue by hjkkkl** [hjkkkl/hjkkkl.github.io#5]
<!doctype  html>               
<html><head><meta charset="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/>
  <style>   [class*="col-"] { float:left; padding:10px;} /* for mobile */
  [class*="col-"] {  width: 100%; }
  @media only screen and (min-width: 768px) {        /* For desktop: */
    .col-1{ width:8.33%;} .col-2{width:16.66%;} .col-3{width:25%;} .col-4{ width:33.33%;} 
    .col-5{width:41.66%;} .col-6{width:50%;} .col-7{ width:58.33%;} .col-8{width:66.66%;} 
    .col-9{width:75%;} .col-10{ width:83.33%;} .col-11{width:91.66%;} .col-12{width:100%;}}  </style>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
  <title>조현정 자바 홈페이지</title></head>
<body><header></header><section>
  <h4>formText1.html</h4><p>소스코드</p>
  <picture>
    <source media="(min-width: 768px)" srcset="./3_1_source.jpg"/>
    <img src="./3_1_source.jpg" alt="소스코드" style="width:auto;"/>
  </picture>
  <p>결과화면</p>
  <picture>
    <source media="(min-width: 768px)" srcset="./3_1_result.JPG"/>
    <img src="./3_1_result.JPG" alt="결과화면" style="width:auto;"/>
  </picture>
  
  <h4>formText1.jsp</h4><p>소스코드</p>
  <picture>
    <source media="(min-width: 768px)" srcset="./3_2_sourcejpg."/>
    <img src="./3_2_source.jpg" alt="소스코드" style="width:auto;"/>
  </picture>
  <p>결과화면</p>
  <picture>
    <source media="(min-width: 768px)" srcset="./3_2_result.jpg"/>
    <img src="./3_2_result.jpg" alt="결과화면" style="width:auto;"/>
  </picture>
  
  
</section></body></html>
